### ISSUE TITLE
Can i put this on an ubuntu ppa?

### ISSUE THREAD

**Issue by Witherking25** [rofl0r/motor#7]

**Comment by rofl0r** [comment]
what does that mean

**Comment by Witherking25** [comment]
An Ubuntu ppa is a personal package archive. It’s basically an Ubuntu repository that is not part of the main repository where people can host their packages

**Comment by rofl0r** [comment]
certainly, this is free software after all. if you have any improvements that don't involve merging back stuff we deliberately removed, feel free to file PRs.

**Comment by Witherking25** [comment]
Sorry I had an old version

**Comment by Witherking25** [comment]
Also, if I can get this into the official Debian repository, ill do that

**Comment by Witherking25** [comment]
Should I use my own pgp key, or do you have one you want me to use?

**Comment by rofl0r** [comment]
> Should I use my own pgp key

your own, thanks

**Comment by Witherking25** [comment]
many bugs need to be fixed first. building is broken.

**Comment by Witherking25** [comment]
rpmbuild is also broken and is outdated

**Comment by Witherking25** [comment]
and support for debhelper would be cool

**Comment by rofl0r** [comment]
> many bugs need to be fixed first. building is broken.

1.) learn how to describe a bug with details and a proper bug report
2.) open an issue
3.) fix it, or get it fixed
4.) profit

**Comment by Witherking25** [comment]
can it go into the official debian repository instead?

**Comment by rofl0r** [comment]
> can it go into the official debian repository instead?

i don't think so since this repo is anything but official, but you can of course try, but don't expect me to assume any responsibilities or maintenance work for this.

**Comment by Witherking25** [comment]
ill maintain it.
i'm currently waiting for someone to sponsor the package.
can you merge the pr?
#11

**Comment by Witherking25** [comment]
there are literally billions of packages in the official repository that are as "official" as this one.

**Comment by Witherking25** [comment]
it will also trickle down into ubuntu and raspbian and kali and other debian based distributions
